README.md

```diff
@@ -30,5 +75,13 @@ Maintainers
 License
 -------
 
-This code is licensed under the BSD 2-Clause license. Please see the LICENSE
-file for more information.
+This code is licensed under the BSD 2-Clause license. Please see the file
+[LICENSE][7] for more information.
+
+[1]: http://caml.inria.fr/ocaml/
+[2]: https://opam.ocaml.org/
+[3]: https://github.com/ocaml/opam-dockerfiles
+[4]: https://github.com/xapi-project/xs-opam
+[5]: https://github.com/xapi-project/sm/blob/master/drivers/XE_SR_ERRORCODES.xml
+[6]: https://github.com/xapi-project/xen-api/blob/master/scripts/examples/python/XenAPI.py
```

Comment:
I totally agree. Overall, the scripts folder in xen-api could use  some general tidying up.